twisted/web/_http2.py

```diff
@@ -577,8 +577,13 @@ def _handleWindowUpdate(self, event):
         streamID = event.stream_id
 
         if streamID:
-            # Update applies only to a specific stream. If we don't have the
-            # stream, that's ok: just ignore it.
+            # Update applies only to a specific stream. We may have already
```

Comment:
I think that the `Update applies only to a specific stream.` part is not adding any new information.